### PR TITLE
create_wrapped_native_account() implemented

### PR DESCRIPTION
**Pull request by kuznetsov-m**

### CODE REVIEW

**Comment by michaelhly** [comment]
@kuznetsov-m thank you for contributing! But to merge in your PRs, please run `make lint` and `make format` locally so that your PRs will pass the CI tests. Thank you!

**Comment by kuznetsov-m** [comment]
@michaelhly Thanks for the advice! I didn't really execute these commands.
During the execution of the `make lint` command and received the following error: 
```
...
pipenv run mypy --namespace-packages --explicit-package-bases spl
Loading .env environment variables...
usage: mypy [-h] [-v] [-V] [more options; see below]
            [-m MODULE] [-p PACKAGE] [-c PROGRAM_TEXT] [files ...]
mypy: error: unrecognized arguments: --explicit-package-bases
make: *** [lint] Error 2
```
Could you help me with this error?

**Comment by michaelhly** [comment]
> @michaelhly Thanks for the advice! I didn't really execute these commands.
> During the execution of the `make lint` (`master` branch) command and received the following error:
> 
> ```
> ...
> pipenv run mypy --namespace-packages --explicit-package-bases spl
> Loading .env environment variables...
> usage: mypy [-h] [-v] [-V] [more options; see below]
>             [-m MODULE] [-p PACKAGE] [-c PROGRAM_TEXT] [files ...]
> mypy: error: unrecognized arguments: --explicit-package-bases
> make: *** [lint] Error 2
> ```
> 
> Could you help me with this error?

What version of `mypy` do you have installed?

**Comment by kuznetsov-m** [comment]
> > @michaelhly Thanks for the advice! I didn't really execute these commands.
> > During the execution of the `make lint` (`master` branch) command and received the following error:
> > ```
> > ...
> > pipenv run mypy --namespace-packages --explicit-package-bases spl
> > Loading .env environment variables...
> > usage: mypy [-h] [-v] [-V] [more options; see below]
> >             [-m MODULE] [-p PACKAGE] [-c PROGRAM_TEXT] [files ...]
> > mypy: error: unrecognized arguments: --explicit-package-bases
> > make: *** [lint] Error 2
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > Could you help me with this error?
> 
> What version of `mypy` do you have installed?

Environment installed from Pipfile.lock
`pipenv install --dev`

https://github.com/michaelhly/solana-py/blob/36a9eaf5051feecf79a770f02f53cccd241b09db/Pipfile.lock#L702

```
mypy                          0.782      
mypy-extensions               0.4.3
```

**Comment by michaelhly** [comment]
> 0.782

ahh sorry about this. I think you need to be on these versions:
```

**Comment by michaelhly** [comment]
> > > @michaelhly Thanks for the advice! I didn't really execute these commands.
> > > During the execution of the `make lint` (`master` branch) command and received the following error:
> > > ```
> > > ...
> > > pipenv run mypy --namespace-packages --explicit-package-bases spl
> > > Loading .env environment variables...
> > > usage: mypy [-h] [-v] [-V] [more options; see below]
> > >             [-m MODULE] [-p PACKAGE] [-c PROGRAM_TEXT] [files ...]
> > > mypy: error: unrecognized arguments: --explicit-package-bases
> > > make: *** [lint] Error 2
> > > ```
> > > 
> > > 
> > >     
> > >       
> > >     
> > > 
> > >       
> > >     
> > > 
> > >     
> > >   
> > > Could you help me with this error?
> > 
> > 
> > What version of `mypy` do you have installed?
> 
> Environment installed from Pipfile.lock
> `pipenv install --dev`
> 
> https://github.com/michaelhly/solana-py/blob/36a9eaf5051feecf79a770f02f53cccd241b09db/Pipfile.lock#L702
> 
> ```
> mypy                          0.782      
> mypy-extensions               0.4.3
> ```
> 
> I read in the documentation about [--namespace-packages](https://mypy.readthedocs.io/en/stable/command_line.html#cmdoption-mypy-namespace-packages) and [--explicit-package-bases](https://mypy.readthedocs.io/en/stable/command_line.html#cmdoption-mypy-explicit-package-bases) flags. Therefore, these arguments must be known to `mypy`. I seem to be doing something wrong.

Can you try pulling in the latest laster, run `rm -rf Pipfile.lock` and then run `pipenv install --dev` again?

**Comment by kuznetsov-m** [comment]
> > > > @michaelhly Thanks for the advice! I didn't really execute these commands.
> > > > During the execution of the `make lint` (`master` branch) command and received the following error:
> > > > ```
> > > > ...
> > > > pipenv run mypy --namespace-packages --explicit-package-bases spl
> > > > Loading .env environment variables...
> > > > usage: mypy [-h] [-v] [-V] [more options; see below]
> > > >             [-m MODULE] [-p PACKAGE] [-c PROGRAM_TEXT] [files ...]
> > > > mypy: error: unrecognized arguments: --explicit-package-bases
> > > > make: *** [lint] Error 2
> > > > ```
> > > > 
> > > > 
> > > >     
> > > >       
> > > >     
> > > > 
> > > >       
> > > >     
> > > > 
> > > >     
> > > >   
> > > > Could you help me with this error?
> > > 
> > > 
> > > What version of `mypy` do you have installed?
> > 
> > 
> > Environment installed from Pipfile.lock
> > `pipenv install --dev`
> > https://github.com/michaelhly/solana-py/blob/36a9eaf5051feecf79a770f02f53cccd241b09db/Pipfile.lock#L702
> > 
> > ```
> > mypy                          0.782      
> > mypy-extensions               0.4.3
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > I read in the documentation about [--namespace-packages](https://mypy.readthedocs.io/en/stable/command_line.html#cmdoption-mypy-namespace-packages) and [--explicit-package-bases](https://mypy.readthedocs.io/en/stable/command_line.html#cmdoption-mypy-explicit-package-bases) flags. Therefore, these arguments must be known to `mypy`. I seem to be doing something wrong.
> 
> Can you try pulling in the latest master, run `rm -rf Pipfile.lock` and then run `pipenv install --dev` again?

Now it works great! Thank you!
`make lint` and `make format` do not require any modifications to this PR. Is there anything else to improve this PR?

**Comment by michaelhly** [comment]
@kuznetsov-m are you sure? CI says you need to run `make format`.

**Comment by kuznetsov-m** [comment]
@michaelhly really sorry, I missed changes.

**Comment by kuznetsov-m** [comment]
@michaelhly It seems I managed to pass all the checks. Are there any more recommendations for edits?

File: spl/token/client.py [hunk 1]
```diff
@@ -283,7 +284,40 @@ def create_wrapped_native_account(
...
+
+        txn.add(
+            spl_token.initialize_account(
+                spl_token.InitializeAccountParams(
+                    account=new_account.public_key(), mint=WRAPPED_SOL_MINT, owner=owner, program_id=program_id
+                )
+            )
+        )
+
+        conn.send_transaction(txn, payer, new_account, opts=TxOpts(skip_preflight=True, skip_confirmation=False))
```
Comment (michaelhly) [inline spl/token/client.py:318]: Let's not hard code `skip_confirmation` and make this an optional parameter on the function.

**Review by michaelhly — APPROVED** [review]

**Pull request merged**